src/ddmd/backend/backconfig.c

```diff
@@ -51,7 +51,7 @@ void out_config_init(
                         // 2: fake it with C symbolic debug info
         bool alwaysframe,       // always create standard function frame
         bool stackstomp,        // add stack stomping code
-        bool avx,               // use AVX instruction set
+        unsigned char avx,      // use AVX instruction set (0, 1, 2)
```

Comment:
I know, but can't we C++11's syntax  (same as in D): `enum name : type { enumerator = constexpr , enumerator = constexpr , ... }` 
(http://en.cppreference.com/w/cpp/language/enum)